### PR TITLE
fix: Integration test close function again

### PR DESCRIPTION
**Pull request by danieljbruce**
This fixes an issue with the close function where promises of promises were once returned, but now we simply return a promise so that when we use `await` then the code is executed properly.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -930,7 +930,7 @@ export class Bigtable {
  * that a callback is omitted.
  */
 promisifyAll(Bigtable, {
-  exclude: ['instance', 'operation', 'request'],
+  exclude: ['close', 'instance', 'operation', 'request'],
```
Comment (danieljbruce) [inline src/index.ts:933]: In the absence of this change, the test just seems to hang forever.

**Review by danieljbruce — COMMENTED** [review]
A comment to clarify the original problem.

File: src/index.ts (hunk 1 above)
Comment (bcoe) [reply src/index.ts:937]: `promisifyAll` turns a `cb` based function into a promise based function, it looks like close was already a promise, and did not expect a `callback` as an argument:

```js
close(): Promise<void[]>
```

**Review by bcoe — APPROVED** [review]

File: src/index.ts (hunk 1 above)
Comment (danieljbruce) [reply src/index.ts:937]: Yes. I am making sure the reader of this PR knows why this change is necessary.

**Pull request merged**